lib/cortex/snippets/client.rb

```diff
@@ -21,6 +21,16 @@ def snippet(request, options = {}, block)
         end
       end
 
+      def snippet_content(request, options = {}, content)
```

Comment:
Agreed I am closing this PR since I created a `Snippet` component to take in whatever the `snippet` methods returns.